nixos/modules/services/networking/gdomap.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.gdomap;
+in
+{
+  #
+  # interface
+  #
+  options = {
+    services.gdomap = {
+      enable = mkOption {
+        default = false;
+	description = "
+	  Whether to enable gdomap, the GNUstep distributed objects daemon.
+
+	  Note that gdomap runs as root.
+        ";
+      };
+
+      pidfile = mkOption {
```

Comment:
I also wonder whether it'd be possible to make the daemon remain in the foreground instead?